packages/smooth_app/lib/pages/preferences_v2/app_bars/app_bar_constanst.dart

```diff
@@ -0,0 +1,6 @@
+const double LOGGED_IN_APP_BAR_EXPANDED_HEIGHT = 310.0;
+const double LOGGED_OUT_APP_BAR_EXPANDED_HEIGHT = 268.0;
+const double TOOLBAR_HEIGHT = 92.0;
```

Comment:
Have you checked that your AppBar fits well when a text multiplier is applied?

Reply:
I've tested and made some adjustments. Here are the screenshots for each iOS text multiplier

![Simulator Screenshot - iPhone 16 Plus - 2025-05-21 at 10 02 13](https://github.com/user-attachments/assets/d9de0aba-3851-4870-8c78-599156b4e40e)
![Simulator Screenshot - iPhone 16 Plus - 2025-05-21 at 10 02 19](https://github.com/user-attachments/assets/eaffa685-a55b-4aac-9f16-31e9b436136d)
![Simulator Screenshot - iPhone 16 Plus - 2025-05-21 at 10 03 12](https://github.com/user-attachments/assets/3177e334-68da-47ef-9aeb-7af99a3c085b)
![Simulator Screenshot - iPhone 16 Plus - 2025-05-21 at 10 03 03](https://github.com/user-attachments/assets/5286aa6b-54d3-4044-9b2c-fde7f4e4923f)